### ISSUE TITLE
modify LeftAtrium use case to use itk directly rather than simpleitk

### ISSUE THREAD

**Issue by cchriste** [SCIInstitute/ShapeWorks#301]

**Comment by prafulag** [comment]
I would recommend staying with sitk unless there are missing api's which we need. Sitk is more consistent in image I/O. Itk manipulates the orientation differently based on numpy array flags.

**Comment by cchriste** [comment]
Very interesting.
The bigger issue is that we'll want to simply use shapeworks.Image.read() rather than itk directly, so it might be worth identifying the underlying issues that differ between the two.

**Comment by prafulag** [comment]
sure, the issue is numpy array flags  C_CONTIGUOUS (C)  and F_CONTIGUOUS (F) [Read Only Flags]

They identify how the array is saved/constructed. I am not aware of any ways to overwrite these flags, which then results in rotated matrices. Surprisingly, sitk read image function does not get affected by these flags.
This is all I know.

**Comment by AtefehKashani** [comment]
As far as I remember from our last discussion about this, we had this issue
of reading files in numpy arrays without changing the read data and Praful
suggested using simpleitk, based on his experience during interneship. I
will dig more into this to see hoe I can use itk directly without causing
new issues.

**Comment by cchriste** [comment]
It is possible that SimpleITK is making a copy of the array, which would be less efficient, and that seems to be its signature trait. We might have to be the same if we're supposed to be returning column-major arrays. One way to find out is just to look at the SimpleITK read image function. It's all public source.

**Comment by AtefehKashani** [comment]
I have checked the imread function of itk which reads the input images. If this function manipulates the orientation of the input image, it is not random, and since we are using argmin function to find the median file, it won't cause any problem for us. Having this in mind, I have changed the GroomUtils to use itk for reading input images , and now we can get rid of the SimpleITK in conda_install. We also should make sure that ITK is built with python support.
